docs/md/dev-wgs.md

```diff
@@ -1,14 +1,16 @@
 # ExternalWholeGenomeReprocessing workload
 
+## Inputs
 An `ExternalWholeGenomeReprocessing` workload
-requires the following inputs
-for each workflow in the workload.
+specifies the following inputs
+for each workflow:
 
-- input_cram
-- sample_name
+- input_cram (required)
```

Comment:
What if we're reprocessing a bam, not a cram?

Reply:
So it looks like that field is just named in a confusing way. If you pass in a `.bam` file, the module will use that as the `input_bam` to the workflow otherwise  if it's a `.cram` file that becomes the `input_cram`.

Reply:
Hmm. `input_aligned_file` might be a better name then.

Reply:
Thanks for the suggestion @tlangs! I think that's a clearer name as well. I'll make a ticket & then discuss with the rest of the team probably at our next refinement meeting.